packages/ui-components/src/components/Matrix/MatrixCell.tsx

```diff
@@ -36,6 +38,12 @@ const DataCellContent = styled.div<{
     $characterLength > 30
       ? '25ch'
       : '13ch'}; // Apply the min width to the content because the cell has padding and we want the content to have a min width and then the padding on top of that
+
+  // If cell is in an expandable row, show nothing in its empty state (instead of the default em
+  // dash). This ‘parent’ class comes from {@link MatrixRow}.
+  .parent &.${EMPTY_CELL_CLASS} {
```

Comment:
I feel I should either:

- pop into `MatrixRow` and `export const MATRIX_ROW_CLASS_PARENT = 'parent'`, or
- just use the literal string `'empty'` here

Any opinions? I can see you’re using the `.parent` class directly in styled-components, and I kind of like the simplicity—even if it’s theoretically less sound

Reply:
You can for sure export from `MatrixRow`, this was likely just an oversight from me

Reply:
Just checking - our expandable rows can be nested multiple levels (I think in reality 2 is the most we have - check UNFPA for some examples), does this handle these?

Reply:
It didn’t handle those—thanks for mentioning it! Third time’s a charm 🤞:

![Screenshot 2024-04-18T151641](https://github.com/beyondessential/tupaia/assets/33956381/61159df3-6da8-40de-b7ac-4edb2d8f0293)

Reply:
Actually, hold off on reviewing because I think I have a way to only hide it if it’s a category **and** there’s no data

Reply:
Good to go. Sorry about all the notifications/emails!